### PR TITLE
Manage order hearing screen changes

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/ComplexTypes/ManageOrders/FL400/ComplexTypes.json [hunk 1]
```diff
@@ -682,6 +682,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "20/08/2021",
+    "ID": "OtherDraftOrderDetails",
+    "ListElementCode": "additionalRequirementsForHearingReq",
+    "FieldType": "Text",
```
Comment (sahaSonali) [inline definitions/private-law/json/ComplexTypes/ManageOrders/FL400/ComplexTypes.json:689]: Use TextArea

File: definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json [hunk 2]
```diff
@@ -35,6 +35,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "20/08/2021",
+    "ID": "OtherOrderDetails",
+    "ListElementCode": "additionalRequirementsForHearingReq",
+    "FieldType": "Text",
+    "ElementLabel": "Additional requirements for hearing request",
```
Comment (sahaSonali) [inline definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json:43]: Use TextArea

File: definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json [hunk 3]
```diff
@@ -970,5 +970,32 @@
     "SecurityClassification": "Public",
     "Searchable": "N",
     "FieldShowCondition": "isCafcassCymru=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/09/2023",
+    "ID": "HearingData",
+    "ListElementCode": "hearingOptions3Label",
```
Comment (sahaSonali) [inline definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json:977]: Change fields name

File: definitions/private-law/json/ComplexTypes/ManageOrders/FL400/ComplexTypes.json (hunk 1 above)
Comment (channa-patil) [reply definitions/private-law/json/ComplexTypes/ManageOrders/FL400/ComplexTypes.json]: Done

**Comment by shashisk97** [comment]
Closing and reopening the PR as Pods are not getting cleared.

**Review by yogendraU — APPROVED** [review]

**Pull request merged**